packages/app-mobile/root.tsx

```diff
@@ -783,6 +788,7 @@ class AppComponent extends React.Component {
 	public componentWillUnmount() {
 		AppState.removeEventListener('change', this.onAppStateChange_);
 		Linking.removeEventListener('url', this.handleOpenURL_);
+		this.screenWidthChangeEvent.remove();
```

Comment:
`if (this.screenWidthChangeEvent) this.screenWidthChangeEvent.remove(); this.screenWidthChangeEvent = null`

Reply:
Also have a look at the naming convention for other such variables. Call it `unsubscribeScreenWidthChangeHandler_`